test/match-test.js

```diff
@@ -615,6 +615,79 @@ describe("sinonMatch", function () {
         });
     });
 
+    describe(".every", function () {
+        it("throws if given argument is not a matcher", function () {
+            assert.exception(function () {
+                sinonMatch.every({});
+            }, "TypeError");
+            assert.exception(function () {
+                sinonMatch.every(123);
+            }, "TypeError");
+            assert.exception(function () {
+                sinonMatch.every("123");
+            }, "TypeError");
+        });
+
+        it("returns matcher", function () {
+            var every = sinonMatch.every(sinonMatch.any);
+
+            assert(sinonMatch.isMatcher(every));
+        });
+
+        it("wraps the given matcher message with an \"every()\"", function () {
+            var every = sinonMatch.every(sinonMatch.number);
+
+            assert.equals(every.toString(), "every(typeOf(\"number\"))");
+        });
+
+        it("fails to match anything that is not an object or an iterable", function () {
+            var every = sinonMatch.every(sinonMatch.any);
+
+            assert.isFalse(every.test(1));
```

Comment:
Instead of using `assert.isFalse`, you can use [`refute`](http://docs.busterjs.org/en/latest/modules/referee/#refute)... you just have to import it from `referee` at the top of the file.